<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>892</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#MainWindow {
    background: #606060
}

QPushButton {
    border: none;
    padding: 0.25em;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #0055ff, stop: 1 #0040e1);
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #2075ff, stop: 1 #2620ff);
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget {
    background: #606060
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="6,9,2">
      <item>
       <layout class="QVBoxLayout" name="plotLayout" stretch="1,1">
        <item>
         <widget class="QwtPlot" name="inputPlot">
          <property name="canvasBackground" stdset="0">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QwtPlot" name="outputPlot">
          <property name="canvasBackground" stdset="0">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="modelStackWidget">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QStackedWidget#modelStackWidget {
    margin: 2.5em;
}</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="angularSimPage">
         <property name="styleSheet">
          <string notr="true">QWidget#angularSimPage {
    border: 2px solid #EEEEEE;
}</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="AngularSim" name="angularSim" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="linearSimPage">
         <property name="styleSheet">
          <string notr="true">QWidget#linearSimPage {
    border: 2px solid #EEEEEE;
}</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="LinearSim" name="linearSim" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="modelFormLayout">
        <property name="labelAlignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="formAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <item row="4" column="0">
         <widget class="QLabel" name="pLabel">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>P:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="pSpinBox">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-99.998999999999995</double>
          </property>
          <property name="maximum">
           <double>99.998999999999995</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="iLabel">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>I:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QDoubleSpinBox" name="iSpinBox">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-99.998999999999995</double>
          </property>
          <property name="maximum">
           <double>99.998999999999995</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="dLabel">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>D:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QDoubleSpinBox" name="dSpinBox">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-99.998999999999995</double>
          </property>
          <property name="maximum">
           <double>99.998999999999995</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="offsetLabel">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Offset:</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QDoubleSpinBox" name="offsetSpinBox">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="minimum">
           <double>-99.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="modelComboBox">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>Angular</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Linear</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="modelComboLabel">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Model Type: </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="resetButton">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Reset Model</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LinearSim</class>
   <extends>QWidget</extends>
   <header>linearsim.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AngularSim</class>
   <extends>QWidget</extends>
   <header>angularsim.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
